ofctrl/ofSwitch.go

```diff
@@ -228,110 +228,125 @@ func (self *OFSwitch) handleMessages(dpid net.HardwareAddr, msg util.Message) {
 	switch t := msg.(type) {
 	case *common.Header:
 		switch t.Header().Type {
-		case openflow13.Type_Hello:
+		case openflow15.Type_Hello:
 			// Send Hello response
 			h, err := common.NewHello(4)
 			if err != nil {
 				log.Errorf("Error creating hello message")
 			}
 			self.Send(h)
 
-		case openflow13.Type_EchoRequest:
+		case openflow15.Type_EchoRequest:
 			// Send echo reply
-			res := openflow13.NewEchoReply()
+			res := openflow15.NewEchoReply()
 			self.Send(res)
 
-		case openflow13.Type_EchoReply:
+		case openflow15.Type_EchoReply:
 			self.lastUpdate = time.Now()
 
-		case openflow13.Type_FeaturesRequest:
+		case openflow15.Type_FeaturesRequest:
 
-		case openflow13.Type_GetConfigRequest:
+		case openflow15.Type_GetConfigRequest:
 
-		case openflow13.Type_BarrierRequest:
+		case openflow15.Type_BarrierRequest:
 
-		case openflow13.Type_BarrierReply:
+		case openflow15.Type_BarrierReply:
 
 		}
-	case *openflow13.ErrorMsg:
+	case *openflow15.ErrorMsg:
+		/* Get the original message type from the error message data field. */
```

Comment:
To be consisted with other comments, you could use `// comment` instead of `/* commment */`.

Reply:
sure. Done.